### DISCUSSION TITLE
Next dev rebuilding api routers without any changes

### DISCUSSION THREAD

**Discussion by LawJolla** [vercel/next.js#19983] [Help]
Pulling my hair out on this one.  Using the default api graphql starter in `/pages/api/graphql`
(I took it back to the following for troubleshooting purposes)
```
export default function handler(req, res) {
  res.statusCode = 200
  res.setHeader('Content-Type', 'application/json')
  res.end(JSON.stringify({ name: 'John Doe' }))
}
```

My console is:

![Image 2020-12-08 at 2 30 49 PM](https://user-images.githubusercontent.com/17485128/101543914-2a0ceb00-3962-11eb-83a0-714a7934a866.jpg)

Clueless how to troubleshoot further.  Any help is greatly appreciated!

**Comment by borekb** [comment]
This has also been an annoyance for us for a long time. Fortunately, it's now configurable in Next.js via `onDemandEntries`, like this in `next.config.js`:

```js
module.exports = {
  onDemandEntries: {
    maxInactiveAge: 3600 * 1000, // one hour
  },
};
```

See [docs](https://nextjs.org/docs/api-reference/next.config.js/configuring-onDemandEntries).
